test/integration/006_simple_dependency_test/test_simple_dependency.py

```diff
@@ -161,17 +157,18 @@ def models(self):
 
     @property
     def packages_config(self):
-        # dbt-event-logging@0.1.5 requires dbt-utils.git@0.1.12, which the
+        # this revision of dbt-integration-project requires dbt-utils.git@0.5.0, which the
         # package config handling should detect
         return {
             'packages': [
                 {
-                    'git': 'https://github.com/fishtown-analytics/dbt-utils',
-                    'revision': '0.1.12',
+
+                    'git': 'https://github.com/fishtown-analytics/dbt-integration-project',
+                    'revision': 'config-version-2-deps'
                 },
                 {
-                    'git': 'https://github.com/fishtown-analytics/dbt-event-logging.git',
-                    'revision': '0.1.5',
+                    'git': 'https://github.com/fishtown-analytics/dbt-utils.git',
+                    'revision': '0.5.0',
```

Comment:
We released dbt-utils v0.6.0 at the end of last week, which is compatible with dbt v0.18.0 (i.e. uses `adapter.dispatch`). I'm not sure if you want to use that version instead throughout.

Reply:
Unfortunately, because dbt-utils is now using sensible version bounds, upgrading to that is just going to break integration tests as soon as we rev dev/kiyoshi-kuromiya to 0.19.0a1 (which I should probably do).

I think in the long run we're just going to want to make another package that's like dbt-integration-project, but depends upon dbt-integration-project. Tests shouldn't care about "real" packages.